components/job-orchestration/job_orchestration/scheduler/job_config.py

```diff
@@ -22,13 +22,15 @@ class PathsToCompress(BaseModel):
 
 class FsInputConfig(BaseModel):
     type: typing.Literal[InputType.FS.value] = InputType.FS.value
+    dataset: str = "default"
```

Comment:
Can we use a constant for this so that we can share the default value everywhere the PR uses it (especially between compression and search configs)?